pages/Home.md

```diff
@@ -6,9 +6,11 @@ Take a tour of the pages on the left and read ones that you may need.
 
 ## Wayland info (especially useful for Xorg users)
 
-A Wayland compositor is a fully autonomous Display Server, like Xorg itself. 
+A Wayland compositor is a fully autonomous Display Server, like Xorg itself.
 It is **not** possible to mix'n'match Wayland compositors like you could on Xorg
-with window managers and compositors.
+with window managers and compositors. It is also not entirely possible to use
```

Comment:
It's not recommended. Many utility apps like polybar / xrandr / xdotool will not work or cause issues. Wayland equivalents are heavily recommended.

Reply:
Added a note for discouragement.